test/common/http/header_utility_test.cc

```diff
@@ -939,5 +939,84 @@ TEST(ValidateHeaders, ContentLength) {
   EXPECT_TRUE(should_close_connection);
 }
 
+// NET-NEW 1.22
+// These tests were added in the changes being backported from 1.22
```

Comment:
ec042c15cf331157a51b15c8fe1af16fc14f2611
Make best-effort attempt to make this test compile